tests/cql/ValueLiteralsAndSelectors.xml

```diff
@@ -126,135 +126,135 @@
 	<group name="Decimal">
 		<test name="DecimalZero">
 			<expression>0.0</expression>
-			<output>42.0-42.0</output>
+			<output>0.0</output>
 		</test>
 		<test name="DecimalPosZero">
 			<expression>+0.0</expression>
-			<output>42.0-42.0</output>
+			<output>0.0</output>
 		</test>
 		<test name="DecimalNegZero">
 			<expression>-0.0</expression>
-			<output>42.0-42.0</output>
+			<output>-0.0</output>
```

Comment:
shouldn't this just be zero for the output?

Reply:
Yep, fixed